examples/docusaurus/README.md

```diff
@@ -4,6 +4,7 @@
 
 This example is built using [Docusaurus](https://docusaurus.io), a modern static website generator, and allows you to add a live cell in the Docusaurus site.
 
+
```

Comment:
This looks useful and like a fairly lightweight integration.  Can you explain the usecase here?  Why is it a good thing to embed a live cell into a Docusaurus site?  Would this work better for internal docs or external facing docs?

For external facing docs, what is the standard way to host a jupyter notebook without huge security risks or exorbitant AWS bills?

Reply:
The very first case would be the documentation of this repo it self with demo pages like https://jupyter-react.datalayer.tech/docs/demos/cell/

Then the external case would be software libraries willing to share demos of what they do (numpy, xarray are doing that).

I guess internal case can also be found to share knowledge with interactive code in companies.

Regarding the cloud costs, an option if to use pyodide (python in the browser) which is currently supported (not documented) by the Notebook component of this repo.